gen/calc.py

```diff
@@ -234,6 +234,11 @@ def calculate_adminrouter_auth_enabled(oauth_enabled):
     return oauth_enabled
 
 
+def validate_os_type(os_type):
+    can_be = ['coreos', 'el7']
+    assert os_type in can_be, 'Must be one of {}. Got {}'.format(can_be, os_type)
```

Comment:
For now we can drop in tin `gen/aws/calc.py` since azure doesn't follow it. We should eventually move it up another level.